source/common/quic/envoy_quic_proof_source.cc

```diff
@@ -16,7 +16,10 @@ namespace Quic {
 quic::QuicReferenceCountedPointer<quic::ProofSource::Chain>
 EnvoyQuicProofSource::GetCertChain(const quic::QuicSocketAddress& server_address,
                                    const quic::QuicSocketAddress& client_address,
-                                   const std::string& hostname) {
+                                   const std::string& hostname, bool* cert_matched_sni) {
+  // TODO(DavidSchinazi) parse the certificate to correctly fill in |cert_matched_sni|.
+  *cert_matched_sni = false;
```

Comment:
What's the impact of setting this to false?

Reply:
There isn't any, the core library simply plumbs this value through (another user needed it) and Envoy doesn't read it on the other side so this is a no-op